### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dan-palmer1**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,25 +14,168 @@ More information can be found in our [Wiki Pages](https://github.com/hydroshare/
...
+
+docker-compose -f local-dev.yml logs
+
+Or
+
+docker logs <container name>
+
+
+
+Warning Suppression
```
Comment (martinseul) [inline README.md:159]: This section about warning suppression can probably be removed as it pertained to a special circumstance for Mark and the document listed is not openly accessible.

File: README.md [hunk 2]
```diff
@@ -14,25 +14,168 @@ More information can be found in our [Wiki Pages](https://github.com/hydroshare/
...
+# when containers are already running in another terminal/shell
+
+docker exec -it hydroshare python manage.py test -v 3 --failfast
+
+
+
+Branching
+When you activate a new branch, just bring the stack down and up again. Sometimes you can get away with a warm restart of the stack or even relying on the Django debug mode (doing nothing but waiting). 
 
 ## Usage
```
Comment (martinseul) [inline README.md:181]: we also have the section about setting up debugging that seems to be missing and should be included, I'm open to adding it here to keep it all together or leaving it as a separate doc:

File: README.md (hunk 1 above)
Comment (dan-palmer1) [reply README.md]: Removed in current commit

File: README.md (hunk 2 above)
Comment (dan-palmer1) [reply README.md:161]: Changed the URL to point to google doc vs github.

**Review by martinseul — APPROVED** [review]

**Pull request merged**